### PR TITLE
fix a verbal mistake

### PR DESCRIPTION
**Pull request by kuramal**
Signed-off-by: Su Xiaolin <linxxnil@126.com>

/kind documentation

**What this PR does / why we need it**:
fix a verbal mistake

### CODE REVIEW

File: docs/proposals/mapper-design.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Mapper is an application that is used to connect and and control devices. Follow
 Mapper can be specific to a protocol where standards are defined i.e Bluetooth, Zigbee, etc or specific to a device if it a custom protocol.
 
 ## Motivation
-All devices can be connected and controller by drivers provided by drivers provided by their vendor.
```
Comment (fisherxu) [inline docs/proposals/mapper-design.md]: connected and controlled？

Comment (kuramal) [reply docs/proposals/mapper-design.md]: @fisherxu yes, it is

**Pull request merged**